microsoft-edge/devtools-guide-chromium/sample-code/sample-code.md

```diff
@@ -153,44 +157,65 @@ This is the source code repo and its directory which stores the files for this d
 
 
 <!-- ====================================================================== -->
-## Download or clone the Demos repo
+## Tools that support cloning a repo and git commit/pull/push operations
```

Comment:
This section feels a bit out of place here. What do you think about moving it as a h4 sub-section inside the "Clone the Edge Demos repo to your drive" section, after "Clone the Edge Demos repo to your drive by using VS Code"?

Reply:
moved